Source/Private/ASImageNode+AnimatedImagePrivate.h

```diff
@@ -15,14 +15,19 @@
 {
   AS::Mutex _displayLinkLock;
   id <ASAnimatedImageProtocol> _animatedImage;
-  BOOL _animatedImagePaused;
   NSString *_animatedImageRunLoopMode;
   CADisplayLink *_displayLink;
   NSUInteger _lastSuccessfulFrameIndex;
   
   //accessed on main thread only
   CFTimeInterval _playHead;
   NSUInteger _playedLoops;
+
+  struct {
+    unsigned int animatedImagePaused:1;
+    unsigned int cropEnabled:1; // Defaults to YES.
```

Comment:
So this is here not because it's related to animated images but then we can group these together to save memory? Do you mind adding a comment to that effect?

Reply:
Yeah; it's due to this category declaring ivars. I'll add the comment.